internal/client.go

```diff
@@ -336,6 +336,14 @@ type (
 		// RequestId is used to deduplicate requests. It will be autogenerated if not set.
 		ResetWorkflowExecution(ctx context.Context, request *workflowservice.ResetWorkflowExecutionRequest) (*workflowservice.ResetWorkflowExecutionResponse, error)
 
+		// UpdateWorkerBuildIDCompatability allows you to update the worker-build-id based version sets for a particular
+		// task queue. This is used in conjunction with workers who specify their build id and thus opt into the
+		// feature. For more, see: <doc link>
```

Comment:
Do we have docs links? otherwise I would remove or leave a not saying they are coming.